site/docs/src/json/identity-providers/links/create-link-request.json

```diff
@@ -1,4 +1,5 @@
 {
+  "displayName": "jared@someLinkedIdp.com",
```

Comment:
Can we use an email address that preserves our theme? If you are going to use `jared` - you could use `jared@piedpiper.com`.